.github/workflows/alpha.yaml

```diff
@@ -0,0 +1,22 @@
+name: alpha
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+
+  release:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      - uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - run: make docker-image DOCKER_IMAGE=replicated/ekco:alpha
```

Comment:
Should we include the git Sha as version here?

Reply:
I dont believe this is necessary https://github.com/replicatedhq/ekco/blob/c1b0d8113a4c13c0b44e981b10ddc35962fda285/Makefile#L6